packages/dataviews/src/dataviews-layouts/grid/index.tsx

```diff
@@ -102,82 +111,85 @@ function GridItem< Item >( {
 				}
 			} }
 		>
-			<div { ...clickableMediaItemProps }>{ renderedMediaField }</div>
-			<SingleSelectionCheckbox
-				item={ item }
-				selection={ selection }
-				onChangeSelection={ onChangeSelection }
-				getItemId={ getItemId }
-				primaryField={ primaryField }
-				disabled={ ! hasBulkAction }
-			/>
+			{ showMedia && (
```

Comment:
When I don't provide a `mediaField` or set `showMedia: false` (any of the those), I'd expect the media in the grid layout not to be visible:

<img width="1239" alt="Screenshot 2024-12-05 at 17 35 44" src="https://github.com/user-attachments/assets/6e116031-507d-4588-be7d-b129c377b332">

However, what happens now if I do not provide a `mediaField` (and don provide any `showMedia`) is this:

<img width="1239" alt="Screenshot 2024-12-05 at 17 34 00" src="https://github.com/user-attachments/assets/9b3c2139-3eaa-422a-804a-0e0a8ad21bf5">

Reply:
I think it's fixed.